README.md

```diff
@@ -1,24 +1,89 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+## Description
 
-* Ruby version
+In this project, I created a budget website in which users can track their finances and what they're being spent on. 
 
-* System dependencies
+## Screenshots
 
-* Configuration
+![image](https://user-images.githubusercontent.com/74814780/159991267-1dcae1b2-f4a4-4837-be7a-dc4877926c5b.png)
 
-* Database creation
+![image](https://user-images.githubusercontent.com/74814780/159991298-b0889634-e711-4676-89e0-fc7f0d271367.png)
 
-* Database initialization
+[Presentation Video](https://www.loom.com/share/28511a885c3f48cea466eecce85126f5)
 
```

Comment:
- [ ] The presentation was good but you have missed some of the things to show in the presentation.

 -  The video should contain a highlight about a piece of the code that you find interesting or is proud of making.